setup/provinces/te_default.txt

```diff
@@ -87,16 +98,16 @@
 	trade_goods=""
 }
 #
-5971=
-{
-	barbarian_power=0
-	civilization_value=0
-	culture="hunnic"
-	province_rank=""
-	religion="eastern_animism"
-	terrain="impassable_terrain"
-	trade_goods=""
-}
+#5971=

```

Comment:
If you're leaving commented out blocks, I suggest adding a comment with an explanation here. Maybe copy the explanation from the changelog.

Reply:
Removed these, not needed.